packages/block-library/src/group/variations.js

```diff
@@ -4,6 +4,59 @@
 import { __, _x } from '@wordpress/i18n';
 import { group, row, stack, grid } from '@wordpress/icons';
 
+const example = {
```

Comment:
Is this example being used at all? 

I have a row block here and it's using the current index.js example. Stack and grid are the same. 

![CleanShot 2024-07-07 at 12 07 17](https://github.com/WordPress/gutenberg/assets/1813435/89684175-9dad-4ce7-bdd7-bfe40a77fb79)

Reply:
It's used for previewing the different block variations in the block inserter.

Steps 5 - 8 in the test instructions cover this aspect. To save you having to fire up the PR locally you can see the original example with the main block inserter from the 30s mark on in the description's demo video.

Reply:
This might be easier:

https://github.com/user-attachments/assets/2c056d26-9196-4b6a-b6b1-f4c8124ce7d3